src/resources.ts

```diff
@@ -5,4 +5,6 @@ const embed = (dir: string, res: Record<string, { default: Uint8Array }>) =>
 export default [
   // For other file types, the Loader must be configured in the <root>/vite.config.ts.
   embed('lua', import.meta.glob<{ default: Uint8Array }>('lua/*.lua', { eager: true })),
+  embed('tex', import.meta.glob<{ default: Uint8Array }>('textemplate/*.tex', { eager: true })),
```

Comment:
First argument of embed is the directory to releases. It should be `textemplate`?

Reply:
Yeah I had just noticed that. The new commit should fix it.